vars/util.groovy

```diff
@@ -320,3 +320,14 @@ void cleanNode(String containerEngine = '') {
         cloud.cleanContainersAndImages(containerEngine)
     }
 }
+
+def spaceLeft() {
+    dir(env.WORKSPACE) {
```

Comment:
HI @mbiarnes 
I'm not an expert: could you please explain me why that dir(env:WORKSPACE) is needed ?

Reply:
as user jenkins you can't do in a job only things if you're in the WORKSPACE - I tested this and you see the whole machine, the /home/jenkins and the /hoem/jenkins/workspace - this work